<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <exportmacro>SCHNAPPS_CORE_API</exportmacro>
 <class>SCHNAppsWindow</class>
 <widget class="QMainWindow" name="SCHNApps">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1106</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SCHNApps</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1106</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_ToggleControlDock"/>
    <addaction name="action_TogglePluginDock"/>
    <addaction name="separator"/>
    <addaction name="action_Settings"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="action_AboutCGoGN"/>
    <addaction name="action_AboutSCHNApps"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="action_Quit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="action_AboutSCHNApps">
   <property name="text">
    <string>About SCHNApps</string>
   </property>
  </action>
  <action name="action_AboutCGoGN">
   <property name="text">
    <string>About CGoGN</string>
   </property>
  </action>
  <action name="action_ToggleControlDock">
   <property name="text">
    <string>Toggle Control Dock</string>
   </property>
  </action>
  <action name="action_TogglePluginDock">
   <property name="text">
    <string>Toggle Plugins Dock</string>
   </property>
  </action>
  <action name="action_Settings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>SCHNApps</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>552</x>
     <y>398</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
